src/preferences/dialog/dlgprefrecord.cpp

```diff
@@ -92,10 +92,12 @@ DlgPrefRecord::DlgPrefRecord(QWidget* parent, UserSettingsPointer pConfig)
     if (index >= 0) {
         // Set file split size
         comboBoxSplitting->setCurrentIndex(index);
-    }
-    else {
+    } else {
         //Use max RIFF size (4GB) as default index, since usually people don't want to split.
         comboBoxSplitting->setCurrentIndex(4);
+        // Save the value immediately, otherwise RecordingManager would fall back
```

Comment:
The comment is now outdated.

Reply:
Yeah, well, I overlooked that we actually only need to use the same default in both unit.
Saving to the config doesn't hurt though.